integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -139,25 +139,25 @@ public String getFieldValue(CreateFileEntry<FileOutStream> o) {
    *
    * @param fsContext the file system context
    * @param fs Alluxio file system
-   * @param opts options
+   * @param fuseFsOpts options for fuse filesystem
    * @param conf Alluxio configuration
    */
   public AlluxioJniFuseFileSystem(
-      FileSystemContext fsContext, FileSystem fs, FuseMountConfig opts, AlluxioConfiguration conf) {
-    super(Paths.get(opts.getMountPoint()));
-    mFsName = conf.getString(PropertyKey.FUSE_FS_NAME);
+      FileSystemContext fsContext, FileSystem fs, AlluxioFuseFileSystemOpts fuseFsOpts,
+      AlluxioConfiguration conf) {
+    super(Paths.get(fuseFsOpts.getMountPoint()));
+    mFsName = fuseFsOpts.getFsName();
     mFileSystemContext = fsContext;
     mFileSystem = fs;
-    mConf = conf;
-    mAlluxioRootPath = Paths.get(opts.getMountAlluxioPath());
-    mMountPoint = opts.getMountPoint();
+    mAlluxioRootPath = Paths.get(fuseFsOpts.getAlluxioPath());
+    mMountPoint = fuseFsOpts.getMountPoint();
     mFuseShell = new FuseShell(fs, conf);
```

Comment:
why Fuse shell need conf? can we move the things that needed by Fuse shell into the Fusefsopts?

Reply:
if we get everything from fuseOptions, we don't need to pass conf anymore